src/core/model_config.proto

```diff
@@ -530,6 +530,25 @@ message ModelOptimizationPolicy
     repeated Accelerator cpu_execution_accelerator = 2;
   }
 
+  //@@
+  //@@  .. cpp:var:: message IndirectPinnedBufferSetting
+  //@@
+  //@@     Specify whether to use a pinned memory buffer when transferring data
+  //@@     between non-pinned system memory and device memory. i.e. when request
+  //@@     contains input in non-pinned system memory and the model instance
+  //@@     accepts GPU input, the input will be processed by two copies: from
+  //@@     non-pinned system memory to pinned memory, and from pinned memory to
+  //@@     device memory.
```

Comment:
"... GPU memory. Similarly when a request must deliver output from GPU memory to non-pinned system memory, pinned memory will be used.